.travis/test.sh

```diff
@@ -2,20 +2,61 @@
 
 set -ev
 
+# Set up the build environment across Julia versions
+# This portion is the same as the default Travis test script
+if [[ -a .git/shallow ]]; then
+    git fetch --unshallow
+fi
+if [[ -f Project.toml || -f JuliaProject.toml ]]; then
```

Comment:
Is this condition entirely necessary while we don't have a Project.toml? Unrelated, but is there a reason there are two *Project.toml files now?

Reply:
JuliaProject.toml is for when you have some other system that requires Project.toml. If it exists it'll be used instead.

Reply:
No the condition isn't necessary but it seemed simpler to me to duplicate the Travis logic wholesale and not have to remember to update things here when we do add a Project.toml file.